src/android/shakeDetector.java

```diff
@@ -5,33 +5,36 @@
  */
 
 import org.json.JSONArray;
-        import org.json.JSONException;
-        import org.json.JSONObject;
+import org.json.JSONException;
+import org.json.JSONObject;
 
-/*
- * Importing dependencies from the notification plugin
- */
 
-        import de.appplant.cordova.plugin.localnotification.TriggerReceiver;
-        import de.appplant.cordova.plugin.notification.Manager;
+
+import de.appplant.cordova.plugin.localnotification.TriggerReceiver;
+import de.appplant.cordova.plugin.notification.Manager;
 
 /*
  * Importing dependencies from the logger plugin
  */
+import edu.berkeley.eecs.emission.cordova.tracker.Constants;
 import edu.berkeley.eecs.emission.cordova.unifiedlogger.Log;
+import edu.berkeley.eecs.emission.cordova.unifiedlogger.NotificationHelper;
```

Comment:
Why are you using NotificationHelper?
You want to use the notification plugin through the `notifyEvent` function